src/pages/admin/dispute/[id].tsx

```diff
@@ -0,0 +1,140 @@
+import { useRouter } from 'next/router';
+import Loading from '../../../components/Loading';
+import { useContext, useEffect, useState } from 'react';
+import StarterKitContext from '../../../context/starterKit';
+import UserNeedsMoreRights from '../../../components/UserNeedsMoreRights';
+import SingleValueForm from '../../../components/Form/singleValueForm';
+import * as Yup from 'yup';
+import { useConfig } from '../../../hooks/useConfig';
+import TalentLayerArbitrator from '../../../contracts/ABI/TalentLayerArbitrator.json';
+import TalentLayerPlatformID from '../../../contracts/ABI/TalentLayerPlatformID.json';
+import { Container } from '../../../components/newlayout/container';
+import usePlatform from '../../../hooks/usePlatform';
+import { useProvider } from 'wagmi';
+import { ethers } from 'ethers';
+import { useChainId } from '../../../hooks/useChainId';
+import Steps from '../../../components/Steps';
+
+function AdminDispute() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { isAdmin, user } = useContext(StarterKitContext);
+  const config = useConfig();
+  const platform = usePlatform(id as string);
+  const chainId = useChainId();
+  const provider = useProvider({ chainId });
+  const arbitratorContractAddress = platform ? platform.arbitrator : null;
+  const arbitratorContract = arbitratorContractAddress
+    ? new ethers.Contract(arbitratorContractAddress, TalentLayerArbitrator.abi, provider)
+    : null;
+  const [arbitratorPrice, setArbitratorPrice] = useState<number | null>(null);
+  let availableArbitrators: { value: string; label: string }[] = [];
+  const [isLoading, setIsLoading] = useState(true);
+  const [isAdminOfThisPlatform, setIsAdminOfThisPlatform] = useState(false);
+
+  // Handle loading state
+  useEffect(() => {
+    if (isAdmin != null && user != null && platform != null && config != null) {
+      setIsAdminOfThisPlatform(platform?.address === user?.address && isAdmin);
+      setIsLoading(false);
+    }
+  }, [isAdmin, user, platform, config]);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+  if (!user) {
+    return <Steps />;
+  }
+  if (!isLoading && !isAdminOfThisPlatform) {
```

Comment:
- isLoading is already check before
- not really clean to have to repeat these conditions, it makes it complicated to maintain and can lead to one forget. Could be better to have a Common Layout for the page content (not the menu etc) of the admin